inc/contract.class.php

```diff
@@ -47,6 +48,15 @@ class Contract extends CommonDBTM {
    protected $usenotepad               = true;
 
 
+   public function getCloneRelations() :array {
```

Comment:
You should drop the `post_addItem` method or you will clone twice the item.

Reply:
Done, I've dropped the two related places